src/tests/ftest/util/soak_test_base.py

```diff
@@ -483,12 +493,23 @@ def execute_jobs(self, jobs, pools):
         """
         job_script_list = []
         # Update the remote log directories from new loop/pass
-        self.sharedsoaktest_dir = self.sharedsoak_dir + "/pass" + str(self.loop)
+        sharedsoaktest_dir = self.sharedsoak_dir + "/pass" + str(self.loop)
         outputsoaktest_dir = self.outputsoak_dir + "/pass" + str(self.loop)
+        soaktest_dir = self.soak_dir + "/pass" + str(self.loop)
         # Create local avocado log directory for this pass
         os.makedirs(outputsoaktest_dir)
         # Create shared log directory for this pass
-        os.makedirs(self.sharedsoaktest_dir, exist_ok=True)
+        os.makedirs(sharedsoaktest_dir, exist_ok=True)
+        # Create local test log directory for this pass
+        os.makedirs(soaktest_dir)
+        if self.enable_remote_logging:
+            result = run_remote(self.log, self.hostlist_clients, f"mkdir -p {soaktest_dir}")
+            if not result.passed:
+                raise SoakTestError(
+                    f"<<FAILED: log directory not created on clients>>: {result.log_output}")
```

Comment:
Similar to Phil's comment above about `result.log_output`. It's a function:
https://github.com/daos-stack/daos/blob/8318e9353595030a7effa25869ff313f0d17ff24/src/tests/ftest/util/run_utils.py#L241-L249